learn/get-started/write-a-graphql-api-with-ballerina.md

```diff
@@ -416,3 +459,11 @@ The result of this request is the following JSON.
   }
 }
 ```
+
+## Learn more
+To learn more about GraphQL services in Ballerina, see the following.
+
+- [`graphql` library documentation](https://lib.ballerina.io/ballerina/graphql/latest)
+- [Hello World](/learn/by-example/graphql-hello-world.html)
+- [GraphQL to Ballerina](/learn/cli-documentation/graphql/#graphql-to-ballerina)
```

Comment:
Shouldn't we add https://ballerina.io/learn/graphql-client-tool/ too?

Reply:
Added.